Userland/Libraries/LibWeb/Internals/Internals.h

```diff
@@ -17,6 +17,7 @@ class Internals final : public Bindings::PlatformObject {
     virtual ~Internals() override;
 
     void gc();
+    JS::Object* hit_test(double x, double y);
```

Comment:
Can this return `JS::GCPtr<JS::Object>` for clarity?

Reply:
that would be nice but it fails to compile because of some `const_cast` in the generated code if I change it this way.